prow/tide/tide.go

```diff
@@ -720,11 +720,12 @@ func (c *Controller) pickBatch(sp subpool) ([]PullRequest, error) {
 		return nil, err
 	}
 	var res []PullRequest
-	for i, pr := range sp.prs {
+	for i := range sp.prs {
 		// TODO: Make this configurable per subpool.
 		if i == 5 {
 			break
 		}
+		pr := sp.prs[len(sp.prs)-i-1]
```

Comment:
We should document that `sp.prs` needs to be sorted with a comment and update the query to explicitly specify the sort ordering instead of relying on GIthub's default.

If we are going to explicitly specify the sort ordering for the query we might as well just sort by ascending number instead?